src/main/scala/com/scalaSparkSkillValidation/part_1.scala

```diff
@@ -0,0 +1,5 @@
+package com.scalaSparkSkillValidation
+
+object part_1 {
+
```

Comment:
ok, what we are doing here?

Reply:
This file will be removed since it's not needed for the project

Reply:
Unused file removed